optimum/habana/transformers/generation/utils.py

```diff
@@ -577,7 +577,7 @@ def generate(
         if model_kwargs["reduce_recompile"]:
             assert generation_config.bucket_size
         if generation_config.reuse_cache:
-            assert self.config.model_type in ["llama"], "reuse_cache only supported by llama at the moment"
+            assert self.config.model_type in ["llama", "mistral"], "reuse_cache only supported by llama at the moment"
```

Comment:
You forgot to add Mistral to asset message

Reply:
added